### PR TITLE
416 home and profile links in header not readable in hover state

### PR DESCRIPTION
**Pull request by karolina-szymanska**

### CODE REVIEW

File: src/js/ui/header/index.js [hunk 1]
```diff
@@ -55,18 +55,16 @@ export const checkState = () => {
...
-      <a class="nav-link text-white fw-semibold" aria-current="page" href="/pages/user/index.html">Profile</a>
-
+      <a class="nav-link text-white fw-semibold" aria-current="page" href="/pages/user/index.html" id="navItems">Profile</a>
     </li>
     <li class="nav-item">
       <a class="nav-link text-white fw-semibold" href="/pages/listings/index.html" id="navItems">Listings</a>
     </li>
     <li class="nav-item">
-      <a class="btn btn-outline-light text-white fw-semibold" href="#" id="signOut">Log out</a>
+      <a class="btn btn-outline-light text-white fw-semibold" href="#" id="signOut navItems">Log out</a>
```
Comment (jaaneh) [inline src/js/ui/header/index.js:67]: You cannot have multiple ID's on a single element

File: src/js/ui/header/index.js [hunk 2]
```diff
@@ -75,16 +73,16 @@ export const checkState = () => {
...
-      <a class="nav-link text-white" aria-current="page" href="#">Profile</a>
+      <a class="nav-link text-white fw-semibold" aria-current="page" href="#" id="navItems">Profile</a>
     </li>
     <li class="nav-item">
-      <a class="nav-link text-white" href="/pages/listings/index.html" id="navItems">Listings</a>
+      <a class="nav-link text-white fw-semibold" href="/pages/listings/index.html" id="navItems">Listings</a>
     </li>
     <li class="nav-item">
-      <a class="btn btn-outline-light text-white" href="#" id="logOutUser">Log out</a>
+      <a class="btn btn-outline-light text-white fw-semibold" href="#" id="logOutUser navItems">Log out</a>
```
Comment (jaaneh) [inline src/js/ui/header/index.js:85]: You cannot have multiple ID's on a single element

**Review by jaaneh — CHANGES_REQUESTED** [review]
Make sure the HTML you're updating is valid please.

**Comment by karolina-szymanska** [comment]
Closing this PR as it´s causing some unexpected conflicts, which is probably caused by this branch being much behind others. Will open a new issue and PR for this.

**Pull request closed without merging**